wagtail/core/blocks/list_block.py

```diff
@@ -88,6 +88,7 @@ def render_form(self, value, prefix='', errors=None):
             'help_text': getattr(self.meta, 'help_text', None),
             'prefix': prefix,
             'list_members_html': list_members_html,
+            'classname': getattr(self.meta, 'form_classname', None),
```

Comment:
Rather than renaming the attribute to something more generic (classname) I have the gut feeling that it should be renamed to something equally specific (i.e. keep the name `form_classname`) or more specific. The reason is that within the context of the template, there could be many html elements that have a customizable `classname`, so the generic name `classname` could be confusing.

Reply:
I agree but that will go against the convention already set in structBlock by renaming form_classname to classnames when rendering the form. I think it should stay as is.

Reply:
Yes, that makes sense.

Reply:
thanks, yeah I want other devs to be able to override this the same way they would the structBlock, including how the template variables are made available. I'll see who else from the team can review this one now.